app.js

```diff
@@ -10,3 +10,151 @@ document.querySelectorAll('.menu-item').forEach((n) => n.addEventListener('click
+    featured_image: 'image/fitness.png',
+    lang_list: ['javascript', 'html', 'css'],
+    breadcrumbs: ['Home', 'Works', 'Calenda'],
+    source: 'https://github.com/rmjspecial',
+    live_demo: 'https://github.com/rmjspecial/personal-portfolio',
+  },
+};
+
+// reverse the order of work card
+function workCardShouldReverse(index) {
+  if (index % 2 === 0) {
+    return 'row-reverse';
+  }
+  return '';
+}
+
+// create the breadcrumbs
+function breadcrumbsHtml(workObj, key, bread = '') {
+  /* eslint-disable no-restricted-syntax */
+  for (const breadcrumb in workObj[key].breadcrumbs) {
+    if (breadcrumb) {
+      bread += `<li>${workObj[key].breadcrumbs[breadcrumb]}</li>`;
+    }
+  }
+  /* eslint-enable no-restricted-syntax */
+
+  return bread;
+}
+
+// create the language list
+function languageHtml(workObj, key, langList = '') {
+  /* eslint-disable no-restricted-syntax */
+  for (const lang in workObj[key].lang_list) {
+    if (lang) {
+      langList += `<li><span>${workObj[key].lang_list[lang]}</span></li>`;
+    }
+  }
+
+  return langList;
+}
+
+//  select the works container
+const workCardContainer = document.querySelector('.works-grid-container');
+let workCard = '';
+let index = 0;
+
+// create the work cards
+for (const key in works) {
+  if (key) {
+    index += 1;
+    workCard
+      += `
+        <div class="flex works-grid-item ${workCardShouldReverse(index)}">
+          <div class="work-image">
+            <img class="width-100" src="${works[key].featured_image}" alt="${works[key].title}">
+          </div>
+          <div class="flex work-details-wrapper">
+          <h3 class="section-title margin-top-12">${works[key].title}</h3>
+          <div class="work-period"> <ul>${breadcrumbsHtml(works, key)} </ul> </div> 
+          <p class="work-description text-color-primary margin-top-20">${works[key].description}</p>
+          <ul class="work-categories margin-top-12 padding-0">${languageHtml(works, key)}</ul>
+          <button type="button" class="button button-enabled see-project-button" data-work="${key}">See Project</button>
```

Comment:
- [ ] Please fix the icon alignment inside the card buttons and use the correct color for the button text to match the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/174289820-9af8ad89-6a78-4d5f-bd18-a6942af36451.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/174289874-f62c65c4-8a6f-427d-baa3-f8f46c48502c.png)